db/seeds.rb

```diff
@@ -1,7 +1,17 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require "faker"
+
+DatabaseCleaner.clean_with :truncation
+
+images = ['https://upload.wikimedia.org/wikipedia/commons/thumb/c/ca/MaryRose-wooden_spoon1.JPG/1280px-MaryRose-wooden_spoon1.JPG',
+          'http://www.publicdomainpictures.net/pictures/50000/nahled/houten-kunstwerken-011.jpg',
+          'https://upload.wikimedia.org/wikipedia/commons/thumb/f/f6/Bough_bike_Sporty.jpg/913px-Bough_bike_Sporty.jpg']
+
```

Comment:
These images are all different sizes so our row heights are pretty funky. We can do some css in the thumbnail to force these to the right dimensions, or we can edit them to the same aspect ratio.